doc/release-notes-26646.md

```diff
@@ -0,0 +1,8 @@
+JSON-RPC
```

Comment:
26877: we don't have 26646 yet

Reply:
@coderabbitai can you please drop 26877 from this branch and force push? (I don't think you can do this, but maybe you can surprise me!)